backend/src/main/java/reviewme/review/service/module/ReviewMapper.java

```diff
@@ -67,14 +67,30 @@ private void addAnswersByQuestionType(ReviewRegisterRequest request,
             Question question = questionMap.get(answerRequest.questionId());
 
             if (question.getQuestionType() == QuestionType.TEXT) {
-                TextAnswer textAnswer = answerMapper.mapToTextAnswer(answerRequest);
-                textAnswers.add(textAnswer);
+                addIfTextAnswerExists(answerRequest, question, textAnswers);
             }
 
             if (question.getQuestionType() == QuestionType.CHECKBOX) {
-                CheckboxAnswer checkboxAnswer = answerMapper.mapToCheckBoxAnswer(answerRequest);
-                checkboxAnswers.add(checkboxAnswer);
+                addIfCheckBoxAnswerExists(answerRequest, question, checkboxAnswers);
             }
         }
     }
+
+    private void addIfTextAnswerExists(ReviewAnswerRequest answerRequest,
+                                       Question question,
+                                       List<TextAnswer> textAnswers) {
+        if (question.isRequired() || !answerRequest.text().isEmpty()) {
```

Comment:
answerRequest.text()가 null일 경우 NPE가 납니다~ 이 경우도 고려해줘야겠네요!

Reply:
@skylar1220 

사실 지금 프론트에서 '답변하지 않은 텍스트 문항'에 대해서 `""` 라는 문자열을 보내주기 때문에, text가 null일 시나리오는 없긴 합니다. 

그런데 제가 적었지만, 이건 너무 프론트 의존적인 생각인 것 같아요. 이런 맥락을 모르는 다른 개발자가 본다면 이해가 안될 코드일 것 같기도 하고요😅

그러니 `ReviewAnswerRequest`에 이런식의 상수 & 함수를 만들어서  '응답을 안할 때의 서술형 답변'을 검사하게 하는건 어떤가요?

```java
public record ReviewAnswerRequest(

        @NotNull(message = "질문 ID를 입력해주세요.")
        Long questionId,

        @Nullable
        List<Long> selectedOptionIds,

        @Nullable
        String text
) {
    private static final String NOT_ANSWERED_TEXT = "";

    public boolean isTextNotAnswered() {
        return NOT_ANSWERED_TEXT.equals(text);
    }
}
```

```suggestion
if (question.isRequired() || !answerRequest. isTextNotAnswered()) {
```

Reply:
dto로 넘기는 것 최고 좋은 생각 반완!!